qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -218,6 +218,13 @@ def _on_mode_entered(self, mode):
         if mode != usertypes.KeyMode.caret:
             return
 
+        # Clear search, replace with blue selection
```

Comment:
I thought I commented on this before, but apparently not? I think this comment isn't really useful, as we say the same thing below already.